AppCenter/AppCenter/MSLogger.h

```diff
@@ -1,7 +1,6 @@
 #import <Foundation/Foundation.h>
```

Comment:
You can probably move the declaration of `msDefaultLogHandler` to the internal header.

Reply:
This header should also be part of the umbrella header.